src/System/Hatrace.hs

```diff
@@ -630,6 +632,20 @@ data SyscallExitDetails_execve = SyscallExitDetails_execve
   } deriving (Eq, Ord, Show)
 
 
+data SyscallEnterDetails_mmap = SyscallEnterDetails_mmap
+  { address :: Ptr Void
+  , len :: CSize
+  , prot :: MemoryProtectMode
+  , mmapFlags :: MMapMode
```

Comment:
Can we call that `flags`, so that the arguments have the same names as in the `mmap` man page?